### DISCUSSION TITLE
Show last week notes along with the previous workout/day sets during workout

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#126] [Ideas]
We currently show previous workout and previous workout on the same day sets under the exercises during a workout.
Let's also add showing the previous notes for that exercise too.

**Comment by feraask** [comment]
Showing notes from the last session would be a great start!

But ideally we'd be able to scroll through all previous session entries on the same screen where you can enter today's workout values.

There are definitely times when I look back at notes from a while back in the middle of a session, not just notes from the previous session or two.

Would even be fine if you could go to a separate screen with history for a specific exercise but also still have today's sets visible to allow entering a full screen focus for a specific exercise and then returning to the list of today's exercises when done.

**Comment by lsolesen** [comment]
Showing previous notes should be filtered by the exercise label. At least with a toggle - that the app remembers - so you can filter them only to show exercise notes with the same Exercise label.

**Comment by astashov** [comment]
Kinda done, since now we show full history during workout
